Dockerfile

```diff
@@ -38,9 +38,11 @@ ARG REBUILD
 
 RUN yarn install
 
+ADD tests/fixtures/urlSeedFile.txt /app/fixtures/urlSeedFile.txt
```

Comment:
It would be better not to add this to the Dockerfile, since this is just for tests. Can instead mount it as a volume when running the test.